turnpike/config.py

```diff
@@ -17,3 +18,12 @@
 )
 PERMANENT_SESSION_LIFETIME = 60 * 60 * 4
 SESSION_COOKIE_SECURE = True
+
+PLUGIN_CHAIN = ["turnpike.plugins.auth.AuthPlugin", "turnpike.plugins.rh_identity.RHIdentityPlugin"]
```

Comment:
Would it make sense to have these plugin definitions configurable?

Reply:
Created the ability for deployers to override config in a change today.